src/Provider/IslandoraServiceProvider.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace Islandora\Crayfish\Commons\Provider;
+
+use Islandora\Crayfish\Commons\CmdExecuteService;
+use Islandora\Crayfish\Commons\FedoraResourceConverter;
+use Pimple\Container;
+use Pimple\ServiceProviderInterface;
+use Silex\Provider\DoctrineServiceProvider;
+use Silex\Provider\MonologServiceProvider;
+use Silex\Provider\ServiceControllerServiceProvider;
+use Silex\Provider\SecurityServiceProvider;
+use Islandora\Chullo\FedoraApi;
+use Islandora\Crayfish\Commons\Syn\SettingsParser;
+use Islandora\Crayfish\Commons\Syn\JwtAuthenticator;
+use Islandora\Crayfish\Commons\Syn\JwtFactory;
+
+class IslandoraServiceProvider implements ServiceProviderInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function register(Container $container)
+    {
+        // Register services we rely on
+        $container->register(new MonologServiceProvider());
```

Comment:
I think in-service registering is discouraged in Silex. It runs, so not a bug, and yes, container implements `register` method, but this introduces hidden dependencies for IslandoraServiceProvider breaking dependency injection possibilities (There is a lot about the topic in google). The implementer of `IslandoraServiceProvider`, i guess crayfish? should be responsible for registering into `Pimple` those services we rely on.  IslandoraServiceProvider should just issue alerts/throw exceptions in case the named containers are not present/not initialized  plus provide inline documentation stating that it requires those.